src/main/java/edu/usf/cutr/gtfsrtvalidator/api/resource/GtfsRtFeed.java

```diff
@@ -156,8 +157,12 @@ public Response startMonitor(
                 leastInterval< runningTasks.get(sessionModel.getGtfsRtFeedModel().getGtfsRtUrl()).getUpdateInterval()){
             intervalUpdated=true;
         }
+        boolean enableShapes = true;
+        if ("unchecked".equals(enableShapesStr)) {
```

Comment:
Instead of `unchecked`, let's use the value `true`.  I'd like to think ahead to this API being used outside of just our web UI, in which case there may not be a checkbox associated with this value, it may just be set programmatically in code.